runtime/src/bank.rs

```diff
@@ -1222,22 +1222,24 @@ impl Bank {
                 epoch >= leader_schedule_epoch.saturating_sub(MAX_LEADER_SCHEDULE_STAKES)
             });
 
-            let vote_stakes: HashMap<_, _> = self
-                .stakes
-                .read()
-                .unwrap()
-                .vote_accounts()
-                .iter()
-                .map(|(epoch, (stake, _))| (*epoch, *stake))
```

Comment:
`epoch` is obviously misnomer here: https://github.com/solana-labs/solana/blob/a8a77614fe011b199ae0632011a5d9deec9c14f9/runtime/src/stakes.rs#L194